### DISCUSSION TITLE
Ansel build on Windows

### DISCUSSION THREAD

**Discussion by Scorpion078** [aurelienpierreeng/ansel#81] [Q&A] [answered]
A couple days ago building Ansel suddenly failed. Never had a problem before.
I could build it, but when trying to install or creating the Installer I got the error message that cmake_install can't find ca_bundle.crt,
but it says file exists.
I tried again with no luck.
Then I compiled darktable and installed and created the installer with no problems.
After that I compared the cmake_install.cmake files and there DT has adifferent path to the ca_bunde.cert file.
I changed it to C:/msys64/ucrt64/bin/../etc/ssl/certs/ca-bundle.crt
Now everything was working fine again.
Maybe that problem exists only on my PC, but I just wanted to put it out there.

**Comment by aurelienpierre** [comment, marked as answer]
It's fixed now.

**Reply by Scorpion078** [reply]
Thank you

**Comment by marcsitkin** [comment]
Thanks, is it in the new AppImage build?

On Tue, Feb 7, 2023 at 10:04 AM Aurélien PIERRE ***@***.***>
wrote:

> It's fixed now.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/aurelienpierreeng/ansel/discussions/81#discussioncomment-4895281>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AN64KKVLUPP3VNUBL522ZQLWWJXCVANCNFSM6AAAAAAUDASQSE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>


-- 
Marc Sitkin
Mexico 2022 <https://marcsitkin.smugmug.com/Mexico/Mexico-2022/>
50 images from Iceland 2021
<https://marcsitkin.smugmug.com/Iceland/Iceland-2021-50-Images/>
Marc Sitkin Photography Web Site <http://marcsitkin.smugmug.com>
Instagram <https://www.instagram.com/marcsitkin/>
